### DISCUSSION TITLE
Galaxy incorrectly claims TLS Minimum 1.2 support

### DISCUSSION THREAD

**Discussion by awatson1978** [meteor/meteor#12184] [Feature Requests]
Galaxy claims to allow users to disable TLS connections below 1.2.  
<img width="376" alt="Screen Shot 2022-09-15 at 10 06 32 PM" src="https://user-images.githubusercontent.com/675910/190540877-4fad7555-febf-44df-a953-2a2d5200ba75.png">

This is due to vulnerabilities in TLS 1.0 and 1.1, as documented in the following forum thread:
https://forums.meteor.com/t/security-disabling-tls-1-0-and-1-1/58278/10

However, when run against security scanners, it's obvious that TLS 1.0 and 1.1 are still enabled.  
https://www.ssllabs.com/ssltest/analyze.html?d=national%2ddirectory.meteorapp.com&s=52.6.168.70&latest

Termination at the container does not suffice for federal certification.  We need termination at the server.  What is being done to disable TLS 1.0 and 1.1 permanently?

**Comment by aquinoit** [comment]
Hello,

We still have some customers using TLS 1.0, so it hasn't been retired yet, we are already taking steps to update and this will happen soon.

Next week I'll let you know when it will start.

Best Regards,

Philippe Aquino
